app/models/XiEoriAddressInformation.scala

```diff
@@ -18,11 +18,11 @@ package models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class XiEoriAddressInformation(streetNumber1: String,
-                                    streetNumber2: Option[String],
-                                    city: String,
-                                    countryCode: String,
-                                    postalCode: Option[String])
+case class XiEoriAddressInformation(pbeAddressLine1: String,
+                                    pbeAddressLine2: Option[String],
```

Comment:
Could you please update pbeAddressLine2, pbeAddressLine3, pbeAddressLine4 and pbePostCode to have the default value as None and after that we we do not need to specify the value like Some("") or None (if you do not want to use those empty values specifically) at line 40 of XiEoriController.scala, that can be updated to as below

xiEoriInformationRepository.set(eori, XiEoriInformation("", "", XiEoriAddressInformation(""))) or if we want to make it more readable then better to say 

          xiEoriInformationRepository.set(
            eori,
            XiEoriInformation("","", XiEoriAddressInformation(pbeAddressLine1 = ""))

Reply:
Updated default value as None to optional address fields.